### DISCUSSION TITLE
Reply context should render emotes

### DISCUSSION THREAD

**Discussion by PsycloneTM** [Chatterino/chatterino2#3919] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Can't see emotes on reply previews.

### Screenshots

![Screenshot 2022-08-10 222900](https://user-images.githubusercontent.com/27637025/184060636-a6adf533-f59f-4573-953d-b8d2e4a5a00f.png)

### OS and Chatterino Version

Chatterino Nightly 7.3.5 (commit a9c48ad8, 2022-08-07) on Windows 11

**Comment by Felanbird** [comment]
It's intended, it would be awful to implement NaM and various other emotes.

**Comment by jupjohn** [comment]
I believe this was intended behaviour, but I agree that having emotes in that preview would be nice UX-wise

**Comment by dnsge** [comment]
Additionally, this is how Twitch's web chat behaves:

![image](https://user-images.githubusercontent.com/24928223/184063187-97cee129-3767-4bea-ae16-52da23977a82.png)

Not that we have to copy every aspect of web chat, but that was my other reason for the choice.

**Comment by leon-richardt** [comment]
Note that scaling down emotes to the height of the "inline thread root" is technically possible, albeit a little fiddly in the current architecture. (I implemented an inelegant proof of concept a while ago.)

**Comment by pajlada** [comment]
I converted this into a discussion until we reach some consensus. I don't personally care either way, so I'd be fine just leaving it as is rather than adding more complexity to the feature at this point.
